powerspikegg/gateway/BUILD

```diff
@@ -0,0 +1,76 @@
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library", "scala_binary", "scala_test")
+
+filegroup(
+    name = "src",
+    srcs = glob([
+        "src/**/*",
+    ]),
+)
+
+scala_binary(
+    name = "server",
+    srcs = ["src"],
```

Comment:
While doing a reference to another rule, it is recommended to keep the colon. So "src" becomes ":src". Here and on other rules. 

>(It is a matter of convention that the colon is omitted for files, but retained for rules, but it is not otherwise significant.)